src/modules/Telemetry/Sensor/VoltageSensor.h

```diff
@@ -8,6 +8,7 @@ class VoltageSensor
 {
   public:
     virtual uint16_t getBusVoltageMv() = 0;
+    virtual int16_t getCurrentMa() = 0;
```

Comment:
Just guessing, but rather than actually trying to figure out what's up with RAK9154, see if you can get away with:
```cpp
virtual int16_t getCurrentMa() { return 0; }
```

Reply:
Never mind, looks like you got it with that last commit!